uint32 id
std_msgs/String utterance
